Docs/1911 Added number field folder and usage content (#2018)
sl-design-system/components: website/src/categories/components/number-field/usage.md

---
title: Number Field usage
tags: usage
eleventyNavigation:
  parent: Number Field
  key: NumberFieldUsage
---
<section class="no-heading">

<div class="ds-example">
<sl-number-field aria-label="Number field" step-buttons="end" value="100"></sl-number-field>
</div>

<div class="ds-code">

  ```html
<sl-number-field aria-label="Number field" step-buttons="end" value="100"></sl-number-field>
  ```

</div>
</section>

<section>

## When to use

> ### Easy Adjusting Values

Reviewer: ```suggestion
### Easily adjust values
```

Reviewer: Not sure the title describes really describes the use cases where you want to use a number field with buttons.

Maybe something like "For values in a small range, with clear intervals"

Author: This is the main point of the number field. It's mainly for numeric inputs, not just small ranges or intervals.

I adjusted the words according to your suggestion.

> Stepper controls allow users to increment or decrement a number field value in defined steps, ensuring precision and ease when selecting quantities within a set range. They are ideal for contexts like adjusting product amounts, setting measurement values, or tuning numeric settings where direct typing might lead to errors.

Reviewer: ```suggestion
The stepper control buttons allow users to increment or decrement the value in defined steps, within a well defined, preferably small, range. They are ideal for scenarios like adjusting product amounts, setting measurement values, or tuning numeric settings where direct typing might lead to errors.
```

Author: I changed

> ### Contextual Formatting Input

Reviewer: ```suggestion
### For values that require formatting or units```

Author: I changed to "Require Formatting Values". I'm keeping the headlines as straightforward as possible.

Number fields can display format-specific information, such as currency symbols or percentage signs, alongside the input value. This visual formatting helps users recognise the context of the data they enter, improving clarity and reducing mistakes, especially in financial or statistical inputs.

</section>

<section>

## When not to use

> ### Use a Dropdown for Fixed Options

Reviewer: ```suggestion
### When there are a fixed number of options
```

Author: Changed to "For a Fixed Number of Options"
